python/paddle/distributed/fleet/fleet.py

```diff
@@ -362,6 +363,244 @@ def init(
                 )
         return self
 
+    # test allreduce perf
```

Comment:
it is better to remove the annotation? because there is no any other additional information transmission

Reply:
> it is better to remove the annotation? because there is no any other additional information transmission

will fix next PR, thanks